src/bosh-director/spec/unit/bosh/director/api/controllers/deployments_controller_spec.rb

```diff
@@ -1485,6 +1485,69 @@ def manifest_with_errand(deployment_name='errand')
               end
             end
 
+            it 'returns ip addresses with prefix if prefix differs from 32 or 128 for a vm' do
+              9.times do |i|
+                instance_params = {
+                  'deployment_id' => deployment.id,
+                  'job' => "job-#{i}",
+                  'index' => i,
+                  'state' => 'started',
+                  'uuid' => "instance-#{i}",
+                  'variable_set_id' => (Models::VariableSet.create(deployment: deployment).id)
+                }
+
+                instance_params['availability_zone'] = 'az0' if i == 0
+                instance_params['availability_zone'] = 'az1' if i == 1
+                instance = Models::Instance.create(instance_params)
+                2.times do |j|
+                  vm_params = {
+                    'agent_id' => "agent-#{i}-#{j}",
+                    'cid' => "cid-#{i}-#{j}",
+                    'instance_id' => instance.id,
+                    'created_at' => time,
+                  }
+
+                  vm = Models::Vm.create(vm_params)
+
+                  if j == 0
+                    instance.active_vm = vm
+                  end
+
+                  ip_addresses_params = {
+                    'instance_id' => instance.id,
+                    'task_id' => i.to_s,
+                    'address_str' => ("1.2.#{i}.#{j*16}/28").to_s,
```

Comment:
```suggestion
                    'address_str' => "1.2.#{i}.#{j*16}/28",
```
Lint/RedundantTypeConversion